it/uk/gov/hmrc/partnershipidentificationfrontend/connectors/RegistrationConnectorISpec.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.partnershipidentificationfrontend.connectors
 
```

Comment:
As a general comment on the integration tests it looks like a number of tests in the RegistrationControllerISpec class need the call for companies house data to be stubbed.